src/m365/pa/commands/app/app-get.ts

```diff
@@ -80,7 +98,7 @@ class PaAppGetCommand extends PowerAppsCommand {
     try {
       if (args.options.name) {
         const requestOptions: CliRequestOptions = {
-          url: `${this.resource}/providers/Microsoft.PowerApps/apps/${formatting.encodeQueryParameter(args.options.name)}?api-version=2016-11-01`,
+          url: `${this.resource}/providers/Microsoft.PowerApps${args.options.asAdmin ? '/scopes/admin' : ''}${args.options.environmentName ? '/environments/' + formatting.encodeQueryParameter(args.options.environmentName) : ''}/apps/${formatting.encodeQueryParameter(args.options.name)}?api-version=2016-11-01`,
```

Comment:
Currently, it's not possible to get an app by display name as admin. Let's make this work as well.

Reply:
Yes, it was missing. Fixed.